dcap/sys/types/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 #![doc = include_str!("../README.md")]
 #![no_std]
 #![allow(
+    clippy::incorrect_clone_impl_on_copy_type,
```

Comment:
:facepalm: I don't know why I didn't attempt this earlier instead of letting the beta failures persist